### PR TITLE
SONAR-6547 Clear rules overloaded debt

### PR DESCRIPTION
**Pull request by julienlancelot**
SONAR-6547 Clear rules overloaded debt

### CODE REVIEW

**Comment by julienlancelot** [comment]
FYI, build fails with Java 8, should be done with Java 6/7

**Comment by julienlancelot** [comment]
FYI, build fails with Java 8, should be done with Java 6/7

File: server/sonar-server/src/main/java/org/sonar/server/platform/ServerComponents.java [hunk 1]
```diff
@@ -20,6 +20,9 @@
 package org.sonar.server.platform;
 
 import com.google.common.collect.Lists;
+import java.util.Arrays;
```
Comment (simonbrandhof) [inline server/sonar-server/src/main/java/org/sonar/server/platform/ServerComponents.java]: lots of changes on imports. Is that expected ?

**Comment by simonbrandhof** [comment]
LGTM

File: server/sonar-server/src/main/java/org/sonar/server/platform/ServerComponents.java (hunk 1 above)
Comment (julienlancelot) [inline server/sonar-server/src/main/java/org/sonar/server/platform/ServerComponents.java]: It's because of changes in configuration on import

**Pull request merged**